ci/Jenkinsfile.tests

```diff
@@ -62,7 +62,16 @@ pipeline {
           steps {
             sh """#!/bin/bash
               set -eo pipefail
-              make test 2>&1 | tee -a ${LOG_FILE}
+              # set CHILD_CONCURRENCY to 1 to avoid node-gyp build error when running yarn install in parallel
+              CHILD_CONCURRENCY=1 make test 2>&1 | tee -a ${LOG_FILE}
```

Comment:
Use `enviornment` section above:
```Groovy
      environment {
        /* Avoid node-gyp build error when running yarn install in parallel. */
        CHILD_CONCURRENCY = '1'
      }
```
Why repeat the same thing that the line after the comment already states?

Reply:
https://ci.status.im/blue/organizations/jenkins/status-mobile%2Fprs%2Ftests/detail/PR-15071/18/pipeline/39
failed again
`../modules/react-native-status/nodejs/status.cpp:1936:1: fatal error: opening dependency file ./Release/.deps/Release/obj.target/status_nodejs_addon/modules/react-native-status/nodejs/status.o.d.raw: No such file or directory`

Reply:
Also, not sure why we need to run `yarn install` in the target itself, but if that's absolutely necessary and it causes issues just drop the parallel part and let them run sequentially, should be fine if they don't take longer than builds.

Reply:
here https://github.com/status-im/status-mobile/pull/15071#issuecomment-1429621434

also tried `--ignore-scripts`, but component test relies on this to build `@react-native-community/blur`

Reply:
And seems to me like the failure isn't due to concurrently running `yarn install` commands:
```
../modules/react-native-status/nodejs/status.cpp:1936:1: fatal error: opening dependency file ./Release/.deps/Release/obj.target/status_nodejs_addon/modules/react-native-status/nodejs/status.o.d.raw: No such file or directory
```
https://ci.status.im/blue/organizations/jenkins/status-mobile%2Fprs%2Ftests/detail/PR-15071/20/pipeline

Reply:
> Also, not sure why we need to run `yarn install` in the target itself, but if that's absolutely necessary and it causes issues just drop the parallel part and let them run sequentially, should be fine if they don't take longer than builds.

it's used to build the status-go Node.js binding
https://github.com/status-im/status-mobile/commit/e4cba0f6636b60c5d19c6a2003e7433bcb6312d0#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R299

Reply:
Hm, worked locally for me just fine.

Reply:
Ok, seems we don't need `yarn install` for component tests, since it's testing UI components

Reply:
Right, well, if we use Nix shell that should run `yarn install` anyway(which is the main reason why it annoys me that it is part of `packages.json` config) and it will prevent conflicts since it does its won locking.

Reply:
Failed when rerun with `yarn install` removed

can't find `@react-native-community/blur` in component tests

my guess is there is a `prepare` hook in `@react-native-community/blur` package.json, and it somehow do cleanup before rebuild, removes the `lib` folder

try to run sequentially without `yarn install` in component tests

Reply:
@yqrashawn, please ignore the issue with @react-native-community/blur. That is separate and is happening locally. Some changes got merged recently which brought that  issue in and needs to be addressed. In fact you can mute the tests that bring in those problems as it will make it easier to verify the CI is working correctly with all tests passing effectively